tests/bluetooth/bsim_bt/bsim_test_advx/src/main.c

```diff
@@ -777,24 +784,93 @@ static void scan_recv(const struct bt_le_scan_recv_info *info,
 
 	bt_addr_le_to_str(info->addr, le_addr, sizeof(le_addr));
 	printk("[DEVICE]: %s, AD evt type %u, Tx Pwr: %i, RSSI %i %s "
-	       "C:%u S:%u D:%u SR:%u E:%u Prim: %s, Secn: %s "
+	       "C:%u S:%u D:%u SR:%u E:%u I: 0x%x Prim: %s, Secn: %s "
```

Comment:
Minor: Further below you print the interval as `Interval 0x%04x (%u ms)` - Would it make sense to do the same here?

Reply:
Yes, will do.